test/unit/player.test.js

```diff
@@ -1570,6 +1570,21 @@ QUnit.test('should add an audio player region if an audio el is used', function(
   player.dispose();
 });
 
+QUnit.test('should turn on/off audioPosterMode when audioPosterMode is called with true/false', function(assert) {
```

Comment:
suggestion: I think we should separate this into a few distinct tests. Also, in general, I would suggest verifying things by calling `audioPosterMode()` as a getter rather than checking `options_` as we want to test the public API.

- A test that asserts the option defaults to `false`, but can be set to `true` at player creation. 
- A test that asserts calling `audioPosterMode()` as a setter changes the values returned by calling `audioPosterMode()` as a getter.
- A test that asserts that the `vjs-audio-poster-mode` class and tech visibility is changed appropriately when `audioPosterMode()` is called as a setter.

You could do all this in a single test with many assertions, but separating it into multiple tests with a few assertions each is arguably more readable and maintainable. 👍